fastlane_core/spec/analytics/analytics_session_spec.rb

```diff
@@ -46,6 +46,11 @@
 
         session.action_launched(launch_context: launch_context)
 
+        # Modify Fixture if not on Mac
+        unless FastlaneCore::Helper.is_mac?
+          fixture_data[2]['primary_target']['detail'] = `uname`.strip
```

Comment:
Is there a better way to get the correct string (`Linux` e.g.) than shelling out to `uname`?

Reply:
https://stackoverflow.com/a/21082972/865175 ?

Reply:
I didn't want to introduce an additional dependency because of that.

Reply:
We do have `class CPUInspector`, maybe there is something in there that could be used to help identify the correct platform, and we can hardcode platform names based on environment?

Reply:
That didn't pan out, but it made me remember that the Analytics thing actually has to get the data from somewhere itself. This is it:

Reply:
Alternative idea: We could also make this method always return `macOS`, not matter where we run the tests. Then the generated file will match the static fixtures. I think this should work similar to https://github.com/fastlane/fastlane/blob/563ec322458c3e1ecada354b99d893430bea7389/fastlane_core/spec/analytics/analytics_session_spec.rb#L12-L13.